api/OffscreenCanvas.json

```diff
@@ -0,0 +1,639 @@
+{
+  "api": {
+    "OffscreenCanvas": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/OffscreenCanvas",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "60",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "Experimental canvas features"
+              }
+            ]
+          },
+          "chrome_android": {
+            "version_added": null
```

Comment:
This and its members need to be marked as 60 for all versions of Chrome and 47 for both versions of Android. https://www.chromestatus.com/feature/5424182347169792

Reply:
👍

Reply:
Quick question @jpmedley - should I maintain the flag for Chrome versions 60?

ie.

```
          "chrome": {
            "version_added": "60",
            "flags": [
              {
                "type": "preference",
                "name": "Experimental canvas features"
              }
            ]
          },
```

Reply:
Yes. In fact, it needs to be copied to Android, Webview, and both Opera versions.

Reply:
OK. Thanks!

Reply:
@jpmedley I've changed with the latest push. But maybe all associated methods need updating too?